tf/env/local/namespaces.tf

```diff
@@ -0,0 +1,5 @@
+resource "kubernetes_namespace" "api-job-namespace" {
+  metadata {
+    name = "api-jobs"
+  }
+}
```

Comment:
why is this needed for local / not needed for staging & prod?

Reply:
It should also go into staging and prod, I just did not finish this PR yet :)

Reply:
ahh whoops sorry :sweat_smile: